api/store/mongo/migrations/migration_1.go

```diff
@@ -10,12 +10,20 @@ var migration1 = migrate.Migration{
 	Version:     1,
 	Description: "Create the database for the system",
 	Up: func(db *mongo.Database) error {
-		logrus.Info("Applying migration 1 - Up")
+		logrus.WithFields(logrus.Fields{
+			"component": "migration",
+			"version":   1,
+			"action":    "Up",
```

Comment:
Looking at the task description @gustavosbarreto did, he used lowercase for _up_ and _down_. This seems better indeed.